_posts/2024-02-18-how-its-made.md

```diff
@@ -0,0 +1,185 @@
+
+While all this might sometimes appear to be overly pedantic and petty, a strict quality control is important to make sure the game can continue for the next 20 years.
+Everybody has probably noticed that everything, from your desk to the storage shelf in the shed, seems to always descend into chaos unless cleaned up regularly, and code is no exception to this.
+If we wouldn't care about code quality, the source code would probably be a hot mess after 20 years and full of bugs, making any new features really difficult.
+
+One thing of note here is that no author can self-approve a PR, even if they have commit access to OpenTTD.
+In this sense, all PR authors are treated equal and need to follow the same process to get a PR approved.
+
+Finally, if a PR has the coveted approval, it is merged into the main code base.
+From that point in time it is part of the game, but not yet in the version you are playing right now.
+For this it still needs a release, which we'll cover in the next section.
+
+
+## The release process
+
+There are two kinds of releases we make for OpenTTD.
+The first one is the "nightly".
+This is automatically made out of a snapshot of the source code each day.
+The nightly release uses almost the same CI workflow as described for the PR checks, except that the result is published on our website and on Steam.
+
+![Types of versions]({% link /static/img/post_2024-02-18-how-its-made/release_versions.png %}){:width="300px" style="float:right; padding:10px"}
+
+The other kind of releases are the interesting ones, the ones that get a proper version assigned, like the upcoming OpenTTD 14.0.
+These happen whenever we think there are enough new things, improvements, and bug fixes since we made the last major release.
+
+If you've been following the news on OpenTTD, you'll have noticed that we've already released some versions with a 14 in it, but yet we're telling you that 14.0 will come soon.
+You might ask: What's up with that?
+Unfortunately, programmers are just as human as you are, and thus it is very likely that OpenTTD contains numerous bugs at any given time.
+
+To make sure we can give you the best possible game to play, all major releases follow a specific path, which is shown in the image on the right.
+It starts with one or more so-called beta releases.
+A beta release is taken directly from the main source code at a certain point in time.
+The main purpose of beta releases is to gather feedback from players about new features and to find any issues with the game.
+Beta releases are still marked as testing releases and are usually not included in Linux distributions or the normal Steam updates, thus depend on players willingly trying a potentially unstable game version.
+
+Even if it's directly taken from the main source, there's still a lot of stuff that needs to be done even for a beta release.
+Someone has to collect all the changes since the last release and create the changelog from them.
```

Comment:
Suggest: _"look over" all the changes since the last release and "write" the changelog._
Since it almost always involves rephrasing some messages and reordering things, so it has a creative element.